### PR TITLE
fixes the failure in case of running the RSocketClient in node env

### PR DESCRIPTION
**Pull request by OlegDokuka**
This PR provides a fix to the access to `window` variable within the node env in strict mode
which led to failure

### CODE REVIEW

File: packages/rsocket-core/src/RSocketClient.js [hunk 1]
```diff
@@ -114,20 +114,22 @@ export default class RSocketClient<D, M> {
...
+        navigator &&
+        navigator.userAgent &&
+        (navigator.userAgent.includes('Trident') ||
+          navigator.userAgent.includes('Edg'))
+      ) {
+        console.warn(
+          'rsocket-js: Due to a browser bug, Internet Explorer and Edge users may experience WebSocket instability with keepAlive values longer than 30 seconds.',
+        );
+      }
+    } catch (e) {}
```
Comment (stevegury) [inline packages/rsocket-core/src/RSocketClient.js]: Please add a comment explaining why we're swallowing the exception.

Comment (OlegDokuka) [reply packages/rsocket-core/src/RSocketClient.js]: fixed

**Comment by daggerok** [comment]
@OlegDokuka,

issue is still here on 0.0.18

```bash
ReferenceError: window is not defined
    at RSocketClient._checkConfig (/Users/mkostromin/Downloads/code/java-nodejs-rsocket/ts-rsocket-server/node_modules/rsocket-core/build/RSocketClient.js:94:23)
    at new RSocketClient (/Users/mkostromin/Downloads/code/java-nodejs-rsocket/ts-rsocket-server/node_modules/rsocket-core/build/RSocketClient.js:46:10)
    # skipped...
```

_RSocketClient.js_

```js
class RSocketClient {
  constructor(config) {
    this._checkConfig(config); // <-- line RSocketClient.js:46

// ...

  _checkConfig(config) {
    const navigator = window && window.navigator; // <-- line RSocketClient.js:94
```

---
Regards

**Comment by daggerok** [comment]
@OlegDokuka kindly reminder
can you please provide worked project (not just snippets) for 0.0.18 if you have?
Thanks

**Comment by OlegDokuka** [comment]
@daggerok My apologies for the delay. I will try to cut off a release is a closes days.

**Pull request merged**